drivers/sdhc/xlnx_sdhc.c

```diff
@@ -587,14 +587,14 @@ static int xlnx_sdhc_host_props(const struct device *dev, struct sdhc_host_props
 			XLNX_SDHC_SLOT_TYPE_GET);
 	props->host_caps.bus_8_bit_support = XLNX_SDHC_GET_HOST_PROP_BIT(cap,
 			XLNX_SDHC_8BIT_SUPPORT);
-	props->host_caps.bus_4_bit_support = XLNX_SDHC_GET_HOST_PROP_BIT(cap,
+	props->bus_4_bit_support = XLNX_SDHC_GET_HOST_PROP_BIT(cap,
 			XLNX_SDHC_4BIT_SUPPORT);
 
 	if ((cap & CHECK_BITS(XLNX_SDHC_SDR400_SUPPORT)) != 0U) {
-		props->host_caps.hs400_support = (uint8_t)config->hs400_mode;
+		props->hs400_support = (uint8_t)config->hs400_mode;
```

Comment:
```suggestion
		props->hs400_support = config->hs400_mode;
```

both are of type bool